README.md

```diff
@@ -62,22 +57,15 @@ ENVs that have to be set:
 - ROOT_CAMP_UNIT_ID_PTA: The ID of the root Pta-Lager
 
 ### STORAGE
-
 We use ActiveStorage to store files. To set it up in production use these ENV-Variables:
-
 - STORAGE_SERVICE=azure
 - STORAGE_ACCOUNT_NAME=
 - STORAGE_ACCESS_KEY=
 - STORAGE_CONTAINER=
 
-## Tests
-
-docker-compose exec app bin/rspec
```

Comment:
this would be still usable? why deleting it?

Reply:
It is just redundant to line 29, thats why I have deleted it here.